ui/rdb/types.d.ts

```diff
@@ -0,0 +1,4 @@
+declare var acquireVsCodeApi: Function;
+declare var vscode;
```

Comment:
Is there a typescript type we could import for some of these? I thought vscode included typescript definitions? If so, we could do `import type * as vscode from 'vscode'` to make this a little safer.

Reply:
Unfortunately there isn't: https://stackoverflow.com/questions/54135313/webview-extension-in-typescript `VS Code does not include TypeScript typings for the VS Code api available to scripts inside of webviews` we are using vscode not for types but for calling `vscode.postMessage(message);` in extensionIntermediary.